### PR TITLE
OBSDOCS-1280: Logging 5.6.23 Release Notes

### PR DESCRIPTION
**Pull request by shreyasiddhartha**
Change type: Doc update; Logging Z-Stream Release Notes - 5.6.23
Doc JIRA: https://issues.redhat.com/browse/OBSDOCS-1280

Fix Version: 4.12, 4.13

Doc Preview: https://81483--ocpdocs-pr.netlify.app/openshift-enterprise/latest/observability/logging/logging_release_notes/logging-5-6-release-notes.html#cluster-logging-release-notes-5-6-23_logging-5-6-release-notes

SME Review: @periklis 
QE Review: @anpingli 
Peer Review: @ShaunaDiaz

### CODE REVIEW

**Review by periklis — COMMENTED** [review]
LGTM

File: modules/logging-release-notes-5-6-23.adoc [hunk 1]
```diff
@@ -0,0 +1,190 @@
+// module included in logging-5-6-release-notes.adoc
+:_mod-docs-content-type: REFERENCE
+[id="cluster-logging-release-notes-5-6-23_{context}"]
+= Logging 5.6.23
+This release includes a link:https://access.redhat.com/errata/RHBA-2024:6093[OpenShift Logging Bug Fix Release 5.6.23].
+
+[id="openshift-logging-5-6-23-bug-fixes_{context}"]
+== Bug fixes
```
[the remark this answers is not in the document]
Comment (shreyasiddhartha) [reply modules/logging-release-notes-5-6-23.adoc:8]: Yes. It's a practice to use write "None." for Bug fixes and CVEs, if either is nil.

File: modules/logging-release-notes-5-6-23.adoc [hunk 2]
```diff
@@ -0,0 +1,190 @@
+// module included in logging-5-6-release-notes.adoc
+:_mod-docs-content-type: REFERENCE
+[id="cluster-logging-release-notes-5-6-23_{context}"]
+= Logging 5.6.23
+This release includes a link:https://access.redhat.com/errata/RHBA-2024:6093[OpenShift Logging Bug Fix Release 5.6.23].
```
[the remark this answers is not in the document]
Comment (shreyasiddhartha) [reply modules/logging-release-notes-5-6-23.adoc:5]: Removing the article altogether, if that's okay.

**Comment by shreyasiddhartha** [comment]
> @shreyasiddhartha Please prepare a manual CP with conflicts resolved and we can get it merged.

Hi @ShaunaDiaz. Created the PR for 4.12: https://github.com/openshift/openshift-docs/pull/81707

**Pull request merged**